### ISSUE TITLE
xkblayout-state version 1b not detected

### ISSUE THREAD

**Issue by davidak** [repology/repology-rules#297]
```
xkblayout-state: WARNING: possibly ambiguous package name "xkblayout-state-1b", consider adding explicit version
```
Source: https://repology.org/log/1585935

The version number is actually "1b".

https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/xkblayout-state/default.nix#L5
https://github.com/nonpop/xkblayout-state/releases

Maybe you can use the pname and version attributed from nixpkgs directly, so you don't have to split?

It is correctly displayed here: https://repology.org/project/xkblayout-state/versions

**Comment by AMDmi3** [comment]
It is pretty clearly visible on the site that this is parsed correctly. `pname` support has nothing to do with rules, see repology-updater#854.

**Comment by davidak** [comment]
Can i do anything so that this warning disappears? Or is the problem that "1b" is a strange version?

**Comment by AMDmi3** [comment]
No, I'm fixing it within repology/repology-updater#854. The warning fired because it contained letter - though it's not too strange for a version to contain a letter in general, it was worth rechecking in nix due to brokennes of the way names and versions are handled there.
